deepchem/feat/rdkit_descriptors.py

```diff
@@ -6,36 +6,6 @@
 from deepchem.feat.base_classes import MolecularFeaturizer
 
 
-class MolecularWeight(MolecularFeaturizer):
```

Comment:
I think this is safe to delete since it wasn't exposed in the public API `deepchem/feat/__init__.py`. `MolecularWeight` isn't an useful featurizer, and I don't think anyone was actually using it.

@peastman Do you think we're OK to delete this or would it be better to deprecate it instead?

Reply:
If we really think it's not useful, I'd be fine with just removing it.